_pages/home/updating-to-version-2211.35/modernize-apps-migrated-from-6.8-to-2211.19.md

```diff
@@ -1,466 +1,473 @@
 
-The benefits of using the new Angular configuration format are:
-- faster builds: application builds are quicker, making life easier and saving time for developers
-- being future-proof: any new Angular and Spartacus features might require you to use the new configuration format as a prerequisite
+**Note:** You should follow the migration guide here **only if your storefront app was originally created using Spartacus 6.x libraries or older**, and was then updated to 2211.19. If your Spartacus app was originally built with 2211.19 libraries or newer, skip the steps here and go straight to [Update Release 2211.36](./migration.md).
 
-The side-effect consequences of using the new configuration format are:
-- For SSR apps: the server-side rendering and prerendering scripts will be executed differently (it will be explained in the last section of this page "[New commands for SSR projects](#new-commands-for-ssr-projects)")
+## Automatic Migration
 
+Spartacus includes specially-prepared schematics to automatically modernize your app to be as similar as possible to a newly created Angular 17 app.
 
-# Automatic migration
+**Note:** These schematics were only released in `@spartacus/schematics2211.36.0`, but you need to run them before fully upgrading your app to 2211.36. As a result, you need to install `@spartacus/schematics@2211.36.0` in a temporary directory and run the migration schematics from there.
 
-The Spartacus team provides special schematics that automatically modernize the app to look as much as possible like the new Angular 17 apps.
+1. Create a temporary sibling directory for the isolated 2211.36 installation schematics by running the following command from the root directory of your project:
 
-Note: the tool was released only in `@spartacus/schematics` v2211.35.0, but you need to run it before fully upgrading your app to this version.
-To do this, you'll need to install `@spartacus/schematics@2211.35.0` in a temporary directory and execute the migration schematic from there.
+   ```bash
+   node -e "require('fs').mkdirSync('../temp-schematics-36')"
+   ```
 
-Please run those commands from your project root directory:
+2. Install the schematics in the temporary directory by running the following command:
 
-```bash
-# 1. Create a temporary sibling directory for the isolated Schematics v2211.35 installation
-node -e "require('fs').mkdirSync('../temp-schematics-35')"
+   ```bash
+   npm install @spartacus/schematics@2211.36.0 --prefix ../temp-schematics-36
+   ```
 
-# 2. Install schematics in the temporary directory
-npm install @spartacus/schematics@2211.35.0 --prefix ../temp-schematics-35
+3. From the temporary directory, run the schematics on your project by running the following command:
```

Comment:
I'm sorry I wasn't clear initially.
It's crucial to run _all those commands_ from the root directory of the project! (i.e. never change the active directory to the temporary guy)
```suggestion
3. Run the schematics by running the following command from the root directory of your project:
```

Reply:
@Platonn Thank you for clarifying this!
I'll just point out, in the end I decided to add a note at the beginning of the procedure to say that all of the commands should be run from the root directory of your project, rather than including it at the end of every step.